kong/clustering/tls.lua

```diff
@@ -226,7 +228,7 @@ function tls.validate_client_cert(kong_config, cp_cert, dp_cert_pem)
     return nil, err
   end
 
-  return true
+  return cert
```

Comment:
before we were returning `true` no matter what, so this can be a safety check
```suggestion
  return cert or true
```